docs/operator-docs/naming-executions.md

```diff
@@ -26,13 +26,18 @@ In the following excerpt, the `QueueGranules` `config.executionNamePrefix`
 property is set using the value configured in the workflow's
 `meta.executionNamePrefix`.
 
+**Please note**: This `meta` property should not be confused with the rule's `executionNamePrefix` from the previous section.
+
 ### Setting executionNamePrefix config for QueueGranules using rule.meta
 
 If you wanted to use a prefix of "my-prefix", you would create a rule with a `meta` property similar to this:
 
 ```json
-{
-  "executionNamePrefix": "my-prefix"
+"rule": {
```

Comment:
This seems a bit confusing to me. Because above we say that this snippet is just the `meta` property of a rule, but it's now really a fragment of an entire rule? If we want to change the snippet, we should change the language above to be something like, `you should create a rule similar to this`. I have mixed feelings about showing a rule fragment as opposed to a complete rule object

Reply:
The justification for this update is the (demonstrably) confusing context in this example of using the same key as a root rule object, and the snippet *lacking context* re: that.         

If that's concerning, I'd suggest we use a different key in this example altogether to avoid users conflating the rule configuration key with this optional meta value.

Reply:
That said, I take your point re: the language concern - perhaps we could explicitly include a 'snippet' indicator/ellipsis in the example json?

Reply:
Yeah just clarify that it's a rule snippet and including ellipsis seems fine

Reply:
How about 0f974ef15

Reply:
Or, rather 706af1727.    😓

Reply:
Super nit:

```suggestion
{
```

Having the `rule` key here makes it seem like the rule is part of another object

Reply:
Mixed feelings, as I know from prior user feedback *not* having the parent key makes glossing over the *text* super easy for readers who look for concrete <objects> and not <words>. 

That said, the objective is met either way so 👍🏻